onnxruntime/python/tools/kernel_explorer/kernels/vector_add_kernel.cuh

```diff
@@ -50,15 +63,14 @@ __global__ void VectorAddKernel(const T* __restrict__ x,
 }
 
 template <typename T, int ThreadsPerBlock, int VecSize>
-Status LaunchVectorAdd(hipStream_t stream, const T* x, const T* y, T* z, int n) {
-  hipLaunchKernelGGL((VectorAddKernel<T, VecSize>),
-                     dim3(CeilDiv(n, ThreadsPerBlock*VecSize)),
-                     dim3(ThreadsPerBlock),
-                     0, stream,
-                     x, y, z, n);
-  auto status = hipGetLastError();
-  ORT_RETURN_IF(status != hipSuccess, hipGetErrorName(status));
-  return Status::OK();
+Status LaunchVectorAdd(StreamT stream, const T* x, const T* y, T* z, int n) {
+  VectorAddKernel<T, VecSize>
+      <<<dim3(CeilDiv(n, ThreadsPerBlock * VecSize)), dim3(ThreadsPerBlock), 0, stream>>>(x, y, z, n);
```

Comment:
Curious: Does this work fine as-is on ROCM/HIP, or are we hip-ifying these files as part of the build process (sorry, I didn't look too closely at the CMake changes)

Reply:
HIP currently converge it kernel language to CUDA, so they share the same launch syntax now.

NOTE: you should prefer `<<<>>>` kernel launch syntax and `threadIdx` and `blockIdx` than old macros.